modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -102,6 +105,7 @@ spec:
 <12> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
 <13> Specify a snapshot location, unless you use CSI snapshots or a File System Backup (FSB) to back up PVs.
 <14> The snapshot location must be in the same region as the PVs.
+<15> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the snapshot location. If your backup and snapshot locations use different credentials, you create separate profiles in the `credentials-velero` file.
```

Comment:
```suggestion
<15> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the snapshot location. If your backup and snapshot locations use different credentials, create separate profiles in the `credentials-velero` file.
```
Same comment as above. Consider changing to the imperative to parallel the imperative in the first sentence